packages/sdk/src/create/libp2p.ts

```diff
@@ -106,8 +106,13 @@ export async function createLibp2pAndUpdateOptions(
     peerDiscovery.push(...getPeerDiscoveries(pubsubTopics, options.discovery));
   }
 
-  if (options?.bootstrapPeers) {
-    peerDiscovery.push(bootstrap({ list: options.bootstrapPeers }));
+  const bootstrapPeers = [
+    ...(options.bootstrapPeers || []),
+    ...(options.store?.peers || [])
+  ];
+
+  if (bootstrapPeers.length) {
```

Comment:
nit: what if length is 0

Reply:
then it would be skipped, which is expected